client/my-sites/plans-v2/utils.ts

```diff
@@ -65,6 +67,23 @@ export function durationToText( duration: Duration ): TranslateResult {
 		: translate( 'per year' );
 }
 
+/**
+ * Renewal utils.
+ */
+
+export function isEligibleForRenewalAtOldRate( { mostRecentRenewDate }: Purchase ): boolean {
```

Comment:
I'm not sure whether this implementation is the best. I'm not a fan of hardcoding the launch date, which I find risky. Ideally, the returned value should be read directly from the purchase parameter.

Reply:
I would just use the current time and rely on the backend the validate times. Remember, this UI is going live with the reset, so there will never be a point where public users see this UI _after_ the offer reset date.

Reply:
Then I must have misunderstood the logic here. What determines if you're still eligible for a renewal at old rate?

> rely on the backend the validate times

Is that something that needs to be implemented yet?

Reply:
Oh, I see what this is doing. Yeah, unsure of a better way- let's sync with backend folks off here.